frodo.js

```diff
@@ -1,101 +1,106 @@
 /**
```

Comment:
SCRIPT-This hardcoded value might be better suited as a constant at the top of the file.